apis/r/R/SOMADataFrame.R

```diff
@@ -506,8 +516,11 @@ SOMADataFrame <- R6::R6Class(
     #' @param new_domain A named list, keyed by index-column name, with values
     #' being two-element vectors containing the desired lower and upper bounds
     #' for the domain.
-    #' @return No return value
-    change_domain = function(new_domain) {
+    #' @param check_only If true, does not apply the operation, but only reports
+    #' whether it would have succeeded.
+    #' @return No return value if `check_only` is `FALSE`. If `check_only` is `TRUE`,
+    #' returns the empty string if no error is detected, else a description of the error.
```

Comment:
We're invisibly returning the output of `upgrade_or_change_domain()` regardless of `check_only`; either the docs should be changed or we should change what our wrapper is, eg

```r
f <- ifelse(isTRUE(check_only), yes = identity, no = invisible)
return(f(upgrade_or_change_domain(...)))
```